classification2.Rmd

```diff
@@ -52,23 +52,28 @@ the observations in the test set? One way we can do this is to calculate the
 **prediction accuracy**. This is the fraction of examples for which the
 classifier made the correct prediction. To calculate this we divide the number
 of correct predictions by the number of predictions made. 
-This process is illustrated in Figure \@ref(fig:06-ML-paradigm-test).
-Note that there are other measures for how well classifiers perform, such as *precision* and *recall*; 
+
+$$prediction \; accuracy = \frac{number \; of  \; correct  \; predictions}{total \;  number \;  of  \; predictions}$$
```

Comment:
OK to use full words here, but use \mathrm to make these things non-italics.

italics in latex is generally reserved for symbols